redis/cache/redispipebp/baseplate.go

```diff
@@ -59,17 +60,19 @@ func wrapSender(sender redis.Sender, args BaseplateRedisClientArgs) (s redisx.Sy
 	}
 	if len(args.Retry) != 0 {
 		s = MonitoredSync{
-			Sync: s,
-			Name: args.Name + ".attempt",
+			Sync:                s,
+			Name:                args.Name + ".attempt",
```

Comment:
I know this has been here before, but don't we use `_` to separate names in Prometheus?

Reply:
in metrics name yes, but this end up in a label so we don't really care about the format